src/renderer/scss/component/_button.scss

```diff
@@ -1,232 +1,160 @@
+    background-position: center calc(50% + 0.1rem);
+  }
 
-    background-color: $lbry-black;
-    border-radius: var(--btn-radius);
-    color: $lbry-white;
-    opacity: 0.8;
-    padding: 10px;
+  .btn__label {
+    display: none;
   }
+}
 
-  &.btn--header-balance {
-    color: $lbry-teal-5;
-    font-size: 14px;
+.btn--inverse {
+  border: 1px solid $lbry-gray-1;
+  border-radius: 1rem;
+  color: inherit;
+  padding: var(--spacing-vertical-small) var(--spacing-vertical-medium); // 0.5rem 1rem;
+  transition: background-color 0.2s;
 
-    @media only screen and (min-width: $large-breakpoint) {
-      font-size: 18px;
-    }
+  html[data-theme='dark'] & {
+    border-color: rgba($lbry-white, 0.1);
+  }
 
-    .btn__label--balance {
-      color: $lbry-gray-5;
-    }
+  &:not(:hover) {
+    background-color: transparent;
+  }
 
-    &:hover {
-      background-color: transparent;
+  &:hover {
+    background-color: $lbry-gray-1;
 
-      .btn__label--balance {
-        color: $lbry-teal-5;
-      }
+    html[data-theme='dark'] & {
+      background-color: rgba($lbry-white, 0.1);
     }
   }
 
-  &.btn--header-publish {
-    background-color: $lbry-teal-5;
-  }
+  .btn__content {
+    display: flex;
 
-  &.btn--home-nav {
-    background-color: $lbry-white;
-    box-shadow: none;
+    svg {
+      width: 1rem;
+      height: 1rem;
 
-    .btn__content {
-      padding: 0;
+      color: $lbry-gray-4;
+      margin-right: var(--spacing-vertical-small); // 0.25rem;
     }
-  }
 
-  &.btn--inverse {
-    background-color: transparent;
-    box-shadow: none;
-    color: $lbry-teal-5;
+    .btn__label {
+      line-height: 1rem;
+    }
   }
+}
 
-  &.btn--link {
-    margin: 0;
-    padding: 0;
+.btn--load-screen {
+  border-bottom: 1px solid $lbry-white;
+  display: inline-block;
 
-    background-color: inherit;
-    border-radius: 0;
-    box-shadow: none;
-    color: $lbry-teal-5;
-    display: inline;
-    font-size: 1em;
-    min-width: 0;
-    text-align: left;
-    transition: none;
+  &:hover {
+    border-bottom: 1px solid $lbry-blue-1;
+    color: $lbry-blue-1;
   }
+}
 
-  &.btn--no-style {
-    margin: 0;
-    padding: 0;
-
-    background-color: inherit;
-    border-radius: 0;
-    box-shadow: none;
-    color: inherit;
-    font-size: inherit;
-    font-weight: inherit;
-    min-width: 0;
-  }
+.btn--primary {
+  align-self: center; // fixes buttons next to tall elements inside one with `display: flex`
```

Comment:
I think is causing alignment issues. See the transactions table. The `Export` button isn't aligned with the filter dropdown because it's centered vertically. Maybe we could remove this and give buttons a `height`?

Reply:
Eh, it's not a big deal. Your solution is probably better.